src/app/accounts/accounts/accounts.component.ts

```diff
@@ -7,6 +7,8 @@ import { AccountDialogComponent } from "../account-dialog/account-dialog.compone
 
 import { AccountsService } from "../../services/storage/accounts.service";
 import { TransactionsService } from "../../services/storage/transactions.service";
+import { OnDestroy } from "@angular/core/src/metadata/lifecycle_hooks";
```

Comment:
why do you need separate import for it? you can import `OnDestroy` on first row with `OnInit`